Sources/SwiftSyntax/Convenience.swift

```diff
@@ -10,6 +10,33 @@
 //
 //===----------------------------------------------------------------------===//
 
+extension ClosureCaptureSyntax {
+
+  /// Creates a `ClosureCaptureSyntax` with a `name`, and automatically adds an `equal` token to it since the name is non-optional.
```

Comment:
How about:
```suggestion
  /// Creates a ``ClosureCaptureSyntax`` with a `name`, and automatically adds an `equal` token to it since the name is non-optional.
```